app/controllers/mentee_profiles_controller.rb

```diff
@@ -1,6 +1,38 @@
 class MenteeProfilesController < ApplicationController
-	def show
-	end
-	def dashboard
-	end
+  before_action :require_mentee, except: [ :show ]
+  before_action :require_user, only: [:show]
```

Comment:
we already have before_filter for that it's called `require_login` so please call it here and delete your custom method